### ISSUE TITLE
Cannot run from terminal

### ISSUE THREAD

**Issue by antnieszka** [DataMedSci/beprof#35]
```
/beprof/beprof ant6% ➜ python3 curve.py
Traceback (most recent call last):
  File "curve.py", line 7, in <module>
    from beprof import functions
ImportError: No module named 'beprof'
```

and:

```
/beprof ant6% ➜ python3 beprof/curve.py 
Traceback (most recent call last):
  File "beprof/curve.py", line 7, in <module>
    from beprof import functions
ImportError: No module named 'beprof'
```

This also happens when running `profile.py`.

**Comment by grzanka** [comment]
I confirm. Import issues has to be solved. Temporary fix is to adjust PYTHONPATH:

```
# grzanka at gajewski-dell in ~/PycharmProjects/beprof on git:feature/template_update ✖︎ [10:49:55]
→ PYTHONPATH=. python3 beprof/profile.py 

Profile
shape: (4, 2)
X : [0.000,3.000]
Y : [0.000000,2.000000]
Metadata : {'meta': 'data', 'some': 'exemplary'}
 FWHM = 2.000
<class '__main__.Profile'>
X: [0 1 2 3]
Y: [0 1 2 1]
M:  {'meta': 'data', 'some': 'exemplary'}
X: [ 1.5  2.5  3.5  4. ]
Y: [ 1.  1.  2.  1.]
M:  {}

a: 

X:  [ 0.5  1.5  2.   2.5]
Y:  [ 1.  1.  1.  1.]
M:  {'negative': 'one'}
type(diff):  <class '__main__.Profile'>
X: [ 0.   0.5  1.   1.5  2.   2.5  3. ]
Y: [ 0.  -0.5  0.   0.5  1.   0.5  1. ]
M:  {'some': 'exemplary', 'meta': 'data'}

# grzanka at gajewski-dell in ~/PycharmProjects/beprof on git:feature/template_update ✖︎ [10:50:06]
→ python3 beprof/profile.py 
Traceback (most recent call last):
  File "beprof/profile.py", line 1, in <module>
    from beprof import curve
ImportError: No module named 'beprof'

```

**Comment by antnieszka** [comment]
If so, I think this should be included in 0.1.0 milestone.

**Comment by grzanka** [comment]
Does it work from PyCharm ?

**Comment by antnieszka** [comment]
Yes, if I click on `if __name__ == "__main__":` it works:
![](http://i.imgur.com/8EAxFKH.png)

This also works for me: `PYTHONPATH=. python3 beprof/profile.py`

**Comment by grzanka** [comment]
Good, so what we need now i a proper description in the documentation.

In README.md I'd state that beprof is a library and end-users shouldn't "run it".

For developers (there should be developer documentation somewhere in docs) we could say that some files have "main methods" as a tests and to run them they need to do this and that (either run from PyCharm, use PYTHONPATH=. or think about some other smart way to call it).

**Comment by antnieszka** [comment]
Will be fixed in #84
